client/src/pages/SplashScreen.js

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect } from 'react'
+import styled from 'styled-components/macro'
+import { useHistory } from 'react-router-dom'
+
+export default function SplashScreen({ onBoot, onComplete }) {
+  const history = useHistory()
+
+  useEffect(() => onBoot(), [])
+
+  useEffect(() => {
+    setTimeout(() => {
+      onComplete()
+      history.push('/dashboard')
+    }, 4000)
+  }, [history])
+
+  return (
+    <ContentWrapper>
+      <div id="logo">
+        <div id="left"></div>
+        <div id="right"></div>
+      </div>
+      <h1>NintenLog</h1>
+    </ContentWrapper>
+  )
+}
+
+const ContentWrapper = styled.body`
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  background: #fd474b;
+  color: white;
+
+  @keyframes slide_motion {
+    0% {
+      margin-top: -15vmin;
+    }
+    30% {
+      margin-top: -17vmin;
+    }
+    100% {
+      margin-top: 0;
+    }
+  }
+  @-webkit-keyframes slide_motion {
+    0% {
+      margin-top: -15vmin;
+    }
+    20% {
+      margin-top: -12vmin;
+    }
+    100% {
+      margin-top: 0;
+    }
+  }
+  * {
+    outline: none;
+    box-sizing: border-box;
+  }
+
+  #logo {
```

Comment:
Hier ist ja einiges los. Sicher, dass du das Logo über ne id stylen willst? Jerry hat das doch als absolutes No Go gesagt

Reply:
Ja, ist hart gesetzt, ist in dem Fall ok :)